package/Linux.mk

```diff
@@ -98,7 +98,9 @@ install:
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/applications
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/pixmaps
-	$(V1) $(MKDIR) -p $(DESTDIR)$(udevdir)
+# Following line creates udev directory - however udevdir is not previously defined, nor is anything
+# installed to this location in the rest of this recipe
+#	$(V1) $(MKDIR)  $(DESTDIR)$(udevdir)
```

Comment:
Another good catch, this line can just be deleted though. Or if you wanted it would be useful to have it optional to install udev rules. i.e., add the udev rules install if udevdir is defined